hibernate-core/src/main/java/org/hibernate/engine/jdbc/internal/ResultSetReturnImpl.java

```diff
@@ -46,9 +46,9 @@ public ResultSetReturnImpl(JdbcCoordinator jdbcCoordinator, JdbcServices jdbcSer
 	@Override
 	public ResultSet extract(PreparedStatement statement) {
 		// IMPL NOTE : SQL logged by caller
-		long executeStart = 0;
+		long executeStartNanos= 0;
```

Comment:
I guess all the `long executeStartNanos= 0` lines here are missing a space after `executeStartNanos`

Reply:
Thanks for spotting it. I fixed it and rebased it.

Reply:
Saw @Sanne 's comment on your ticket. I don't know too much about it, but it seems another exciting PR from Vlad.